swiftpipeline/argumentparser.py

```diff
@@ -115,6 +119,15 @@ def __setup_parser(self):
             required=True,
         )
 
+        self.parser.add_argument(
```

Comment:
My idea here was to have an API where the names were passed too. I don't like the idea of having positional-only arguments here. I'd prefer it if we could add an argument (with the long name only) and then pass that based on the input yaml. I'm happy to make this change if needed.

Reply:
Not really sure what you mean so please feel free to add how you like after push the next commit.